src/server/terrain/biomes.zig

```diff
@@ -462,17 +464,22 @@ pub const BlockStructure = struct { // MARK: BlockStructure
 		allocator.free(self.structure);
 	}
 
-	pub fn addSubTerranian(self: BlockStructure, chunk: *ServerChunk, startingDepth: i32, minDepth: i32, x: i32, y: i32, seed: *u64) i32 {
+	pub fn addSubTerranian(self: BlockStructure, chunk: *ServerChunk, startingDepth: i32, minDepth: i32, slope: i32, erosion: f32, x: i32, y: i32, seed: *u64) i32 {
 		var depth = startingDepth;
+		var timer = @max(@as(i32, @intFromFloat(@as(f32, @floatFromInt(slope))*erosion)) - 1, 0);
```

Comment:
This has nothing to do with time, so maybe call it `remainingSkippedBlocks` or something

Reply:
Also the `@max` is unnecessary.